src/scales/scale.time.js

```diff
@@ -313,6 +315,26 @@ function generate(min, max, minor, major, capacity, options) {
 		last.add(1, minor);
 	}
 
+	// for series we want to include only user supplied timestamps
+	if (options.distribution === 'series') {
+		time = timestamps[i];
+		if (+moment(time) === +moment(time).startOf(minor)) {
+			ticks.push(time);
+		}
+		for (i = 1; i < timestamps.length; i++) {
+			time = timestamps[i];
+			if (ticks.length === 0) {
+				if (+moment(time).startOf(minor) >= +moment(timestamps[i - 1])) {
```

Comment:
[nit] `+moment(time).startOf(minor)` is used a few times. Might make sense in a local var.

Reply:
done